src/mare/compiler/code_gen/debug_info.cr

```diff
@@ -11,6 +11,7 @@ class Mare::Compiler::CodeGen
       @mod : LLVM::Module,
       @builder : LLVM::Builder,
       @target_data : LLVM::TargetData,
+      @generate_debug_info = true,
```

Comment:
Is this variable used anywhere, or is it replaced by `ctx.options.no_debug`?
```suggestion
```

Reply:
Forgot to remove it